src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
-    background-color: red;
-}
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
+h1 {
+  font-size: 3rem;
+  margin: 1rem 0 3rem 1rem;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-content: center;
+  align-items: center;
+}
+
+.recentScores {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- Hey, let's kindly improve our styles in this area as the button is at the right and the text is at the left and not at the center. see the image of the wireframe to improve your design below:




*your design*
![Screenshot from 2023-04-19 23-45-54](https://user-images.githubusercontent.com/59397159/233216465-d81dc9e6-185e-4b12-bc51-28bbd29accfa.png)

*wireframe*
![Screenshot from 2023-04-19 23-47-05](https://user-images.githubusercontent.com/59397159/233216285-7acfc0ad-87db-4baa-9dba-23d70f936965.png)